.travis.yml

```diff
@@ -16,10 +16,22 @@ virtualenv:
   system_site_packages: true
 install:
   - pip install -r https-everywhere-checker/requirements.txt
-before_script:
-  - sh -e /etc/init.d/xvfb start
+  - pip install -r test/chromium/requirements.txt
 env:
   - DISPLAY=':99.0'
+before_script:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - wget https://chromedriver.storage.googleapis.com/2.18/chromedriver_linux64.zip
+  - unzip chromedriver_linux64.zip
+  - sudo mv chromedriver /usr/bin/chromedriver
+  - sudo apt-get install -y python-software-properties
+  - sudo apt-add-repository "deb http://dl.google.com/linux/chrome/deb/ stable main"
+  - sudo sed -i s/deb-src.*google.*//g /etc/apt/sources.list
+  - wget -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+  - sudo apt-get update -qq
+  - sudo apt-get install -y google-chrome-stable
+  - sudo chmod 1777 /dev/shm
```

Comment:
I've tried, yes. It's disallowed.

There are two concerns regarding that, someone on Travis's issue mentioned that it'll never work without ```sudo```.

Second, the directory where ```chromedriver``` is extracted isn't a PATH directory, and I'm not sure how to move it correctly to the directory where the Selenium Python script exists.

For the first concern, here. https://github.com/travis-ci/travis-ci/issues/3354#issuecomment-90791268. Would you like me to try that?

For the second concern, I might need a little bit of help.